pipeline/notification/README.md

```diff
@@ -0,0 +1,34 @@
+## Description
+
+This Folder is responsible for sending daily email notifications to our users.
+
+
+## Requirements
+The requirements for this folder are:
+- python-dotenv
+- boto3
+- pylint
+
+
+## Environment Variables
+In order to run the scripts you will need the following environment variables, in a *.env* file:
+
+- AWS_KEY
+- AWS_SECRET
+
+
+## The Scripts
+This folder contains the following scripts:
+
+- **alert.py**  
+This script sends SNS alerts to subscribers of topics, based on newly released games.  
+
+   Run from the command line using: 
+  >python3 alert.py
+  
+- **Dockerfile**  
+This script is used to dockerise **alert.py**.  
+  
+  Run from the command line using: 
+  >docker build -t alert .  
```

Comment:
```suggestion
  >docker build -t alert --platform "linux/amd64" .  
```

Reply:
Just realised this says 'run from command line' so feel free to ignore my suggestion!

Reply:
I was thinking maybe we don't need this when running locally?